docs/examples/exporters/GLTFExporter.html

```diff
@@ -73,21 +80,50 @@ <h3>[method:null parse]( [param:Object3D input], [param:Function onCompleted], [
 			<li>onlyVisible - bool. Export only visible objects. Default is true.</li>
 			<li>truncateDrawRange - bool. Export just the attributes within the drawRange, if defined, instead of exporting the whole array. Default is true.</li>
-			<li>binary - bool. Export in binary (.glb) format, returning an ArrayBuffer. Default is false.</li>
-			<li>embedImages - bool. Export with images embedded into the glTF asset. Default is true.</li>
 			<li>animations - Array<[page:AnimationClip AnimationClip]>. List of animations to be included in the export.</li>
 			<li>forceIndices - bool. Generate indices for non-index geometry and export with them. Default is false.</li>
 			<li>forcePowerOfTwoTextures - bool. Export with images resized to POT size. This option works only if embedImages is true. Default is false.</li>
 		</ul>
 		</p>
 		<p>
-		Generates a .gltf (JSON) or .glb (binary) output from the input (Scenes or Objects)
+		Generates a .gltf (JSON) or .glb (binary) output from the input (Scenes or Objects).<br/>
+		The onComplete callback will be called with one argument with a different type depending on the mode: <br />
+		<ul>
+			<li>glb — Blob. The glb's file content.</li>
+			<li>gltf — Object. The chunks object returned by [page:Function parseChunks].</li>
+			<li>embedded — Object. The glTF's JSON content.</li>
+		</ul>
+		</p>
+
+		<h3>[method:null parseChunks]( [param:Object3D input], [param:Function onCompleted], [param:Function onError], [param:Object options] )</h3>
+		<p>
+		[page:Object input] — Scenes or objects to export.<br />
+		[page:Function onCompleted] — Will be called when the export completes.<br />
+		[page:Function onError] — (optional) A function to be called if an error occurs during exporting. The function receives error as an argument.<br />
+		[page:Options options] — Export options see <em>.parse</em><br />
+		</p>
+		<p>
+		The onComplete callback will be called with one argument, an object with the following properties: <br />
+		<ul>
+			<li>json — object. The glTF's JSON content.</li>
+			<li>buffers — Array<[page:Blob Blob]>. Blobs corresponding to the glTF's buffers array. Will be undefined if the buffer does not have a uri or it is not the glb buffer.</li>
+			<li>images — Array<[page:Blob Blob]>. Blobs corresponding to the glTF's images array. Will be undefined if the image does not have a uri.</li>
+		</ul>
+		</p>
+
+		<h3>[method:null createGLBBlob]( [param:Object3D input], [param:Function onCompleted], [param:Function onError] )</h3>
+		<p>
+		[page:Object chunks] — The chunks object returned by [page:Function parseChunks].<br />
+		[page:Function onCompleted] — Will be called when the export completes. The argument will be a blob containing the glb file's content.<br />
+		[page:Function onError] — (optional) A function to be called if an error occurs during exporting. The function receives error as an argument.<br />
```

Comment:
I don't think this method needs to be publicly documented, as users should call `parse` — also note that the input is not an Object3D.